src/components/learner-credit-management/tests/OfferDates.test.jsx

```diff
@@ -16,7 +16,7 @@ describe('<OfferDates />', () => {
 
   describe('with start date only', () => {
     it('date is past', () => {
-      const startDate = moment('2022-01-31');
+      const startDate = dayjs('2022-01-31');
```

Comment:
Personally, I am not a fan of any date libraries that use a single data type to represent both calendar dates and timestamps, which are very different things. Python gets this right but many JS libraries do not.